02-analytics/technical.md

```diff
@@ -108,30 +118,22 @@ deliver additional data.
 
 | Data         | Name        | Comments                                       |
 |--------------|-------------|------------------------------------------------|
-| Name         | name        | A page identifier. See comments and list below |
-| Element Type | elementType | episode, series, ...                           |
-| Element ID   | elementId   | series, episode ID                             |
+| ID | Id | Code of the page if generated or from the list below. String |
+| Title | title | String, native language |
+| Additional info | meta | JSON. Currently unused, but fields can be added as needed |
 
 ### Page/Screen IDs
 
-This are the currently used screens from the app:
-
-* AboutPage
-* AudiencePage
-* CalendarPage
-* ExplorePage
-* InfoPage
-* LiveStreamPage
-* LoginPage
-* NativePlayer
-* ProfileEditPage
-* ProfilePage
-* ProgramPage
-* QueuePage
-* SearchPage
-* SeriesPage
-* SettingsListPage
-* SupportPage
+* about
+* audience
+* calendar
+* liveStream
+* login
+* profileEdit
+* profile
+* search
+* settings
```

Comment:
I thin a general `settings` is more flexible.

Reply:
ok

Reply:
settings with extra property "setting": appLanguage, audioLanguage, subtitleLanguage

Reply:
On web this might be slightly different as we have more space and likely show all options at the same time.

Perhaps settings/profile won't even be a separate page, just a modal, so I might need another event type for that or?

Reply:
I would say that it could still count as a page, and just send maybe "all" for the event type?

Reply:
I agree that it can be a page even though it's just a dropdown or modal

Reply:
I updated the text and are resolving this. Please open a new convo if there is more.